### ISSUE TITLE
About/Service Mesh

### ISSUE THREAD

**Issue by aijamalnk** [aijamalnk/istio.io-design-and-dev#7]
[Netlify link for preview](https://deploy-preview-8985--istio-polidea.netlify.app/latest/about/service-mesh/)

https://drive.google.com/file/d/1SB7VNAdeT4V0dF2PP9mgbYnfqGP3Nwxd/view?usp=sharing

**Comment by sirandrian** [comment]
@wbartosz Add carousel at the bottom of the page

**Comment by aijamalnk** [comment]
This looks good. Just one clarification.

In the carrousel, I see only one dot - perhaps because we only have three items in the carousel. Would we have more dots if we add more items?

![image](https://user-images.githubusercontent.com/14306210/108458052-a6f0f680-7228-11eb-99dd-dd91a6eb0dc6.png)

**Comment by wbartosz** [comment]
Exactly. The solutions carousel is connected with the Solutions in the content directory. The more Solutions you add, the more carousel pages to scroll.
